honeycomb/create_table.py

```diff
@@ -334,16 +334,37 @@ def ctas(select_stmt, table_name, schema=None,
     bucket = schema_to_zone_bucket_map[schema]
     path = validate_table_path(path, table_name)
 
-    full_path = '/'.join([bucket, path]) + '/'
-
-    temp_schema = 'experimental'
-    view_name = '{}_temp_ctas_view'.format(table_name)
-    create_view_stmt = 'CREATE VIEW {}.{} AS {}'.format(
-        temp_schema, view_name, select_stmt)
+    full_path = '/'.join([bucket, path])
 
-    hive.run_lake_query(create_view_stmt)
+    table_rename_template = 'ALTER TABLE {}.{} RENAME TO {}.{}'
+    if '{}.{}'.format(schema, table_name) in select_stmt:
```

Comment:
Question here, what is `honeycomb`'s natural response if we try to write to a table that already exists? Shouldn't it fail?

In other words, shouldn't 

```
if '{}.{}'.format(schema, table_name) in select_stmt:
```

only matter if we specify `overwrite = True`? As in, should the `overwrite` be in the outer loop?

Reply:
You are correct, it should fail. However, since this is just scanning a string, whether `overwrite` is inside or outside of the outer `if` block won't affect anything logically. It would actually make it more difficult to raise the error for attempting to CTAS onto an existing table without `overwrite=True`, so I think it'd be best to leave it as is. Unless you think I'm missing something, in which case I'm all ears!

Reply:
Sounds good to me.